### PR TITLE
updates to session pages: this should be OK for the release of the CfP

### PR DESCRIPTION
**Pull request by simplytalking**
**The CfP date will need to be changed to whenever we publish this!!**

I have updated the pages to to reflect the meeting we held on the 14th October. 
The changes are related to the call for papers area. I have not changed the sponsorship section for example.

I have some errors in my local running version:
[2018-12-05 13:12:55] ERROR `/images/favicon.ico' not found.
[2018-12-05 13:12:55] ERROR `/apple-touch-icon-precomposed.png' not found.
[2018-12-05 13:12:55] ERROR `/apple-touch-icon.png' not found.

Sorry I tried to fix the favicon but so far no success :(

I do not what is causing the apple-touch-icon stuff.

### CODE REVIEW

**Comment by hibri** [comment]
👍  LGTM

File: _config.yml [hunk 1]
```diff
@@ -35,13 +35,13 @@ conference:
...
-  reviewing_deadline: TBC
-  programme_meeting_date: TBC
-  programme_announce_date: TBC
-  shepherding_dates: TBC
+  cfp_deadline_time: 5 PM
+  cfp_early_feedback_deadline: 21st January 2019
+  cfp_close_date: 18th February 2019
+  reviewing_deadline: 7th March 2019
+  programme_meeting_date: 9th March 2019
+  programme_announce_date: 15th March 2019
```
Comment (annashipman) [inline _config.yml]: You should change this date as per discussion in email, maybe to 'mid April'?

File: _includes/lead-a-session-nav.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <ul>
   <li><a{% if page.url == '/lead-a-session.html' %} class="menuactive" {% endif %} href="{{ '/lead-a-session.html' | relative_url }}"><span>Lead a session</span></a></li>
-  <li><a{% if page.url == '/interactivity.html' %} class="menuactive" {% endif %} href="{{ '/interactivity.html' | relative_url }}"><span>How can I make my session interactive?</span></a></li>
-  <li><a{% if page.url == '/submission-stages.html' %} class="menuactive" {% endif %} href="{{ '/submission-stages.html' | relative_url }}"><span>Submission stages</span></a></li>
+  <li><a{% if page.url == '/inter-activity.html' %} class="menuactive" {% endif %} href="{{ '/interactivity.html' | relative_url }}"><span>How can I make my session interactive?</span></a></li>
+  <li><a{% if page.url == '/submission-stages1.html' %} class="menuactive" {% endif %} href="{{ '/submission-stages1.html' | relative_url }}"><span>Submission stages</span></a></li>
```
Comment (annashipman) [inline _includes/lead-a-session-nav.html]: Why have you changed these two page URLs?

File: _includes/prev-conf-programmes-nav.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<ul>
```
Comment (annashipman) [inline _includes/prev-conf-programmes-nav.html]: You don't need to add this, there is already an include for previous conferences - not quite sure what's going on here

File: info-for-speakers.md [hunk 4]
```diff
@@ -1,11 +1,8 @@
 ---
-title: Lead a session
+title: Information for speakers
```
Comment (annashipman) [inline info-for-speakers.md]: This is an internal navigation in a page. If you look at https://www.spaconference.org/spa2018/info-for-speakers.html, this is supports the menu down the left hand side, all under 'Lead a session'. 

If you want to change the way that works, that's fine but please explain why in the commit message.

File: submission-stages1.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: Submission stages
```
Comment (annashipman) [inline submission-stages1.md]: You could just replace the existig submission-stages page - why create a new one?

File: terms-and-conditions.md [hunk 6]
```diff
@@ -36,4 +36,4 @@ layout: page
 <p>To find out what information is held about you or request deletion of all your information, please contact <a href="mailto:admin@spaconference.org">admin@spaconference.org</a>.</p>
 
 <h2>Other</h2>
-<p>The SPA Specialist Group is not responsible for the views or opinions expressed by individual session leaders, contributors to the SPA wiki, or any individuals who are not members of the executive committee.</p>
+<p>The SPA Specialist Group is not responsible for the views or opinions expressed by individual session leaders, contributors to the {{ site.conference.name }} wiki or the {{ site.conference.name }} Slack Workspace, or any individuals who are not members of the executive committee.</p>
```
Comment (annashipman) [inline terms-and-conditions.md]: 👍 Good spot

**Comment by annashipman** [comment]
Duplicate file `favicon.ico1`

**Comment by annashipman** [comment]
I checked it out locally and the 'lead a session' pages are now a little bit confusing because of the lack of internal navigation on the first page. I get what you mean about there being too much information on the page but the internal navigation helps the user understand where they are - it was created by that by a user experience designer who was specifically trying to think about how to make sure the user could get the information they needed.

In terms of making sure the important dates are clear, what about, instead putting the important bits in bold at the top of the main content section? (i.e in the main column, rather than a new column on the right) You don't need the location (it's in the main menu) or the dates of the conference (they are up there under the conference title), so it's just the deadline and the length of sessions, plus an obvious link to submit. For that, I think a button under the nav menu. I've attached a screenshot. If you can't figure out how to do that I can do that later.

I checked it out locally and the 'lead a session' pages are now a little bit confusing because of the lack of internal navigation on the first page. I get what you mean about there being too much information on the page but the internal navigation helps the user understand where they are - it was created by that by a user experience designer who was specifically trying to think about how to make sure the user could get the information they needed.


In terms of making sure the important dates are clear, what about, instead putting the important bits in bold at the top of the main content section? (i.e in the main column, rather than a new column on the right) You don't need the location (it's in the main menu) or the dates of the conference (they are up there under the conference title), so it's just the deadline and the length of sessions, plus an obvious link to submit. For that, I think a button under the nav menu. I've attached a screenshot. If you can't figure out how to do that I can do that later.

Also, it's probably worth replying to comments on the PR itself rather than by email to me so that anyone can help with this and also those coming after you can see what's happened, e.g. how unclear my instructions were! If you just email me then it has to be me who deals with it, and no-one else will ever know what happened... :)

**Comment by annashipman** [comment]
One more thing: the test is failing because html-proofer can't log into that page. There is a way to get html-proofer to ignore specific links, I can't remember right this second how to do it. Need to make sure the check is not failing before we merge.

**Comment by simplytalking** [comment]
which page is failing?

On 9 Dec 2018, at 15:22, Anna Shipman <notifications@github.com<mailto:notifications@github.com>> wrote:


One more thing: the test is failing because html-proofer can't log into that page. There is a way to get html-proofer to ignore specific links, I can't remember right this second how to do it. Need to make sure the check is not failing before we merge.

**Comment by annashipman** [comment]
The Travis check. I’m saying how to fix the red - failing - Travis check. 

--
  Anna Shipman
  www.annashipman.co.uk
  @annashipman

**Comment by simplytalking** [comment]
> Duplicate file `favicon.ico1`

Yes we should get rid of this one - favicon.ico is not working for some reason - it is the standard BCS one and from my limited knowledge should work fine. Not clear why it is being ignored

**Comment by simplytalking** [comment]
> The Travis check. I’m saying how to fix the red - failing - Travis check.

OK I understand now. I leave that in your capable hands :)

**Comment by annashipman** [comment]
Ha! No, I’m suggesting to you how you can fix it!

--
  Anna Shipman
  www.annashipman.co.uk
  @annashipman

**Comment by simplytalking** [comment]
thank God!

On 9 Dec 2018, at 16:02, Anna Shipman <notifications@github.com<mailto:notifications@github.com>> wrote:

Ha! No, I’m suggesting to you how you can fix it!

--
Anna Shipman
www.annashipman.co.uk<http://www.annashipman.co.uk>
@annashipman

**Comment by simplytalking** [comment]
> In terms of making sure the important dates are clear, what about, instead putting the important 
> bits in bold at the top of the main content section? (i.e in the main column, rather than a new 
> column on the right) 

I tried it but you lost the text completely but maybe if we get rid of some of the stuff in the right column as you suggest below it will be OK.

> You don't need the location (it's in the main menu) or the dates of the conference (they are up 
> there under the conference title), so it's just the deadline and the length of sessions, plus an 
> obvious link to submit. For that, I think a button under the nav menu. I've attached a screenshot. If > you can't figure out how to do that I can do that later.

Agreed no location or dates - I will remove.
I have seen one of those submission buttons elsewhere on the site - will try to duplicate - did not see the screenshot as an attachment by the way - am I looking in the wrong place?

> I checked it out locally and the 'lead a session' pages are now a little bit confusing because of the > lack of internal navigation on the first page. I get what you mean about there being too much 
> information on the page but the internal navigation helps the user understand where they are - it 
> was created by that by a user experience designer who was specifically trying to think about how > to make sure the user could get the information they needed.

OK so if we do as you suggest and put the submission button in and put the deadline and session lengths in the main text I think we can try the right hand nav - will do that now.



Also, it's probably worth replying to comments on the PR itself rather than by email to me so that anyone can help with this and also those coming after you can see what's happened, e.g. how unclear my instructions were! If you just email me then it has to be me who deals with it, and no-one else will ever know what happened... :)

So if I had kept everyone on copy the email would have ended up here anyway. But I am now adding replies direct to GitHub

**Comment by simplytalking** [comment]
latest update now changes build.sh to ignore makeproposal.php which hopefully will cause travis to be happy.

**Comment by simplytalking** [comment]
OK fixed the lead-a-session page and put the submit button on the header. Looks better now

File: _includes/lead-a-session-nav.html [hunk 7]
```diff
@@ -1,6 +1,8 @@
 <ul>
   <li><a{% if page.url == '/lead-a-session.html' %} class="menuactive" {% endif %} href="{{ '/lead-a-session.html' | relative_url }}"><span>Lead a session</span></a></li>
-  <li><a{% if page.url == '/interactivity.html' %} class="menuactive" {% endif %} href="{{ '/interactivity.html' | relative_url }}"><span>How can I make my session interactive?</span></a></li>
+  <li><a{% if page.url == '/inter-activity.html' %} class="menuactive" {% endif %} href="{{ '/interactivity.html' | relative_url }}"><span>How can I make my session interactive?</span></a></li>
```
[the remark this answers is not in the document]
Comment (simplytalking) [reply _includes/lead-a-session-nav.html]: OK that has been fixed. thank you for spotting that.

File: _includes/lead-a-session-nav.html [hunk 8]
```diff
@@ -1,6 +1,8 @@
...
   <li><a{% if page.url == '/submission-stages.html' %} class="menuactive" {% endif %} href="{{ '/submission-stages.html' | relative_url }}"><span>Submission stages</span></a></li>
   <li><a{% if page.url == '/info-for-speakers.html' %} class="menuactive" {% endif %} href="{{ '/info-for-speakers.html' | relative_url }}"><span>Information for speakers</span></a></li>
+    <li><a{% if page.url == '/successful-sessions.html' %} class="menuactive" {% endif %} href="{{ '/successful-sessions.html' | relative_url }}"><span>Sample of recent sessions</span></a></li>
+  <li><a{% if page.url == '/prev-conf-programmes.html' %} class="menuactive" {% endif %} href="{{ '/prev-conf-programmes.html' | relative_url }}"><span>Previous conferences programmes</span></a></li>
```
[the remark this answers is not in the document]
Comment (simplytalking) [reply _includes/lead-a-session-nav.html]: Yes I agree that the duplication is horrid - but when applying to a conference I often look at past programmes to see what the conference is like and I just got fed up finding the program in all the old websites. But realistically you are only likely to look at the last four years to get a feel. Would you be OK if I took the last four years of pre-conf-programmes and added it to the bottom of the successful sessions page and then deleted the pre-conf-programmes page? Then removed the pre-conf-programmes from all the navigation etc.

[the remark this answers is not in the document]
Comment (simplytalking) [reply _includes/lead-a-session-nav.html]: OK I just removed the previous-conf-programmes and put the last four years at the bottom of the successful sessions page. Much cleaner :)

File: lead-a-session.md [hunk 9]
```diff
@@ -4,43 +4,53 @@ layout: page
...
 <p>Submissions submitted before {{ site.conference.cfp_deadline_time }} on {{ site.conference.cfp_early_feedback_deadline }} will receive feedback and may be resubmitted.</p>
 <p>Submissions deadline: {{ site.conference.cfp_deadline_time }}, {{ site.conference.cfp_close_date }}</p>
 <p>Programme announced: {{ site.conference.programme_announce_date }}</p>
+<p>If you want to be notified when the important dates are coming up, please register at <a href="https://spaconference.org/scripts/myprofile.php">My SPA</a> and select the option to receive communications by us.</p>
 <p>If you're uncertain about whether to submit your idea, or have other queries, please feel free to get in touch with us at</p>
 <p><em>programme@spaconference.org</em></p>
-<p>Jason Ayers, Programme Chair, {{ site.conference.name_with_year }}</p> -->
+<p>Jason Ayers, Programme Chair, {{ site.conference.name_with_year }}</p>
+</div>
+<!-- <em>Submissions are now closed for SPA Software in Practice 2019.</em> -->
```
[the remark this answers is not in the document]
Comment (simplytalking) [reply lead-a-session.md]: Done. I thought that did not appear on the page - was I wrong?

File: interactivity.md [hunk 10]
```diff
@@ -1,24 +1,29 @@
 ---
-title: Lead a session
+title: How can I make my session interactive?
```
[the remark this answers is not in the document]
Comment (simplytalking) [reply interactivity.md]: I have tidied the navigation within the CfP subsection - they are all the same now and the interactivity page has a better title. I looked at the whole site and all the pages have the page title in the grey bar. I understand that in someway the CfP pages are subpages of the main CfP page but I am not convinced that the extra title in the text and having the grey bar have the same title 'Lead a session' is useful. Have a look at what I have done and if you still feel that it is clearer to revert then I will.

**Comment by annashipman** [comment]
Closed in favour of #68

**Pull request closed without merging**